Source/CCZ4/CCZ4Geometry.hpp

```diff
@@ -64,9 +64,11 @@ class CCZ4Geometry
                 // Trick: For CCZ4, we can add Z terms to ricci by changing
                 // Gamma to chrisvec This way of writing it allows the user to
                 // pass Z/chi = {0};
+                // (MR): I disagree with the above comment
                 ricci_tilde += 0.5 * (vars.h[k][i] * d1.Gamma[k][j] +
                                       vars.h[k][j] * d1.Gamma[k][i]);
-                ricci_tilde += 0.5 * (vars.Gamma[k] - 2 * Z_over_chi[k]) *
+                ricci_tilde += 0.5 * /* (vars.Gamma[k] - 2 * Z_over_chi[k]) **/
+                               chris.contracted[k] *
                                (chris.LLL[i][j][k] + chris.LLL[j][i][k]);
```

Comment:
I'm happy with this suggestion as it means BSSN evolution does not change. I'm still a bit confused why it was coded as
```cpp
(chris.LLL[i][j][k] + chris.LLL[j][i][k])
```
rather than
```cpp
d1.h[i][j][k]
```
even though they're equivalent.

Reply:
I guess my only issue with this is that `ricci_tilde` now explicitly includes Z terms whereas it didn't before.

Reply:
I resolved my qualms by renaming this object to `ricci_hat`.